crates/uv/tests/pip_install.rs

```diff
@@ -1925,95 +1925,51 @@ fn install_only_binary_all_and_no_binary_all() {
               anyio>=3.0.0,<=3.6.2
               anyio>=3.7.0,<=3.7.1
               anyio>=4.0.0
-          and anyio==1.0.0 has no usable wheels and building from source is disabled, we can conclude that any of:
+          and any of:
```

Comment:
Should it be "all of"?

Reply:
Yeah it's a little confusing. I think it's saying "any of a or b" instead of "all of a and b" — we don't use "and" or "or" so "all" and "any" do that. We can probably make this better somehow...